<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>score_ui</class>
 <widget class="QWidget" name="score_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lable_score_1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysourece/picture/kuang.png);
color: rgb(255, 0, 0);
font: 22pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_score_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysourece/picture/kuang.png);
color: rgb(255, 0, 0);
font: 22pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_score_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysourece/picture/kuang.png);
color: rgb(255, 0, 0);
font: 22pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_score_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysourece/picture/kuang.png);
color: rgb(255, 0, 0);
font: 22pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>一营长的意大利炮</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_1_group_win">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>555</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>队名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>胜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>74</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>负</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_1_group_loss">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>70</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>积分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_1_group_score">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_2_group_win">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>北邮第三区交通委</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_2_group_loss">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_2_group_score">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_3_group_win">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>Void</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_3_group_loss">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_3_group_score">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>250</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_4_group_win">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>北邮 第一交通队</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_4_group_loss">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>310</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_4_group_score">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="save_data">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysource/picture/button_background.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="read_data">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysource/picture/button_background.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>读取数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
